.github/workflows/ci.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v3
       with:
-        go-version: '1.18'
+        go-version: '1.22'
```

Comment:
Use the `go-version-file: gomod` directive here. Then we don't have to keep bumping this value